react/panel/components/panelEvents.jsx

```diff
@@ -0,0 +1,153 @@
+var React = require('react');
+
+// Panel Elements
+var PanelEventList = require('./PanelEventList.jsx');
+
+// Inputs
+var InputEditableTextField = require('../inputs/InputEditableTextField.jsx');
+var InputEditableDateTimeField = require('../inputs/InputEditableDateTimeField.jsx');
+var InputCheckbox = require('../inputs/InputCheckbox.jsx');
+var InputFileUpload = require('../inputs/InputFileUpload.jsx');
+var ConfirmationButton = require('../inputs/ConfirmationButton.jsx');
+
+// Bootstrap Elements
+import { Grid, Col, Row, Well, ButtonGroup, Button, Input, DropdownButton, MenuItem } from 'react-bootstrap';
+
+// Misc
+var Dates = require('../../common/Dates.js');
+
+const PanelUserEventsPage = React.createClass({
+	render: function(){
+		return (
+      <div className="panelPage">
+        <Grid>
+          <Row>
+            <Col xs={12} md={12}>
+                <UserEventsList urls={this.props.urls} />
+            </Col>
+          </Row>
+        </Grid>
+      </div>
+		);
+	}
+});
+
+// TODO: Components are currently sharing styles with their show counterparts
+// Update such that they have their own style
```

Comment:
Is this TODO fixed? If not can you make an issue for it instead of just noting it in code?